### PR TITLE
Add info about Swift Package Manager command plugin to README.md

### PR DESCRIPTION
**Pull request by calda**
This PR adds info about the new Swift Package Manager command plugin to the `README.md`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,10 +17,46 @@ Note that brevity is not a primary goal. Code should be made more concise only i
...
   * For rules that don't directly change the format of the code, we should have a lint rule that throws a warning.
   * Exceptions to these rules should be rare and heavily justified.
 
+## Swift Package Manager command plugin
+
+This repo includes a Swift Package Manager command plugin that you can use to automatically reformat or lint your package according to the style guide. To use this command plugin with your package, all you need to do is add this repo as a dependency:
+
+```swift
+dependencies: [
+  .package(url: "https://github.com/airbnb/swift", from: "1.0.0"),
```
Comment (calda) [inline README.md:30]: I'll cut the first release version after merging this PR

File: README.md [hunk 2]
```diff
@@ -17,10 +17,46 @@ Note that brevity is not a primary goal. Code should be made more concise only i
 * This guide is in addition to the official [Swift API Design Guidelines](https://swift.org/documentation/api-design-guidelines/). These rules should not contradict that document.
 * These rules should not fight Xcode's <kbd>^</kbd> + <kbd>I</kbd> indentation behavior.
 * We strive to make every rule lintable:
-  * If a rule changes the format of the code, it needs to be able to be reformatted automatically (either using [SwiftLint](https://github.com/realm/SwiftLint) autocorrect or [SwiftFormat](https://github.com/nicklockwood/SwiftFormat)).
+  * If a rule changes the format of the code, it needs to be able to be reformatted automatically (either using [SwiftFormat](https://github.com/nicklockwood/SwiftFormat) or [SwiftLint](https://github.com/realm/SwiftLint) autocorrect).
```
Comment (calda) [inline README.md:20]: Swapped these since like 90% of our rules are implemented using Swift Format

File: README.md [hunk 3]
```diff
@@ -17,10 +17,46 @@ Note that brevity is not a primary goal. Code should be made more concise only i
...
+
+# When using the Xcode 13 toolchain, or a noninteractive shell, you must use: 
+$ swift package --allow-writing-to-package-directory format
+
+# To just lint without reformatting, you can use `--lint`:
+$ swift package format --lint
+
+# By default the command plugin lints the entire package directory.
+# You can exclude directories using `exclude`:
+$ swift package format --lint --exclude Tests --exclude Samples
```
Comment (calda) [inline README.md:55]:
```suggestion
# By default the command plugin runs on the entire package directory.
# You can exclude directories using `exclude`:
$ swift package format --exclude Tests

# Alternatively you can explicitly list the set of paths and/or SPM targets:
$ swift package format --paths Sources Tests Package.swift
$ swift package format --targets AirbnbSwiftFormatTool
```

**Review by fdiaz — APPROVED** [review]
Great stuff!

**Pull request merged**